Pull_Requests.md

```diff
@@ -0,0 +1,46 @@
+# Code Reviews
+
+## What is a code review?
+When you, or another team member wants add changes to an AlbertaSat repository they will go through a process defined by the (contributing guideline section)[Contributing_README.md]. This typically involves creating a branch, making changes, making a pull request, code review, then the reviewer either asks for changes to be made or approves the pull request. After the pull request is approved the code can be merged to main, or additional changes can be made. In this documentation we will go over guidelines for reviewing and writing pull requests.
+
+## How should a pull request look like?
+__This section we will go over to how write a proper pull request that allows code reviewers to quickly and efficiently review code.__\
+\
+The following are some general rules of what a pull request should look like:\
+- Change the default pull request name from the branch name to a descriptive name summarizes the changes you want to make.
+- Add a useful description that includes the following:
+    - The issue number linked to the pull request.
+    - Any dependent pull requests in a different repository that should be reviewed first.
+    - Describe the changes you made and how the changes you made solve the root issue.
+    - If neccesary, clearly state any feedback you are looking for with a mention to the person who can help with feedback.
+- If your pull request is still a work in progress, prefix the name of the pull request with "[WIP]".
+
+## How should a pull request be reviewed?
+__This section we will go over to how review a pull request.__\
+\
+The following are some general rules of how a reviewer should review a pull request:\
+- If the PR is labelled as "WIP" then it should not be approved.
+- Thoroughly go all changes made and if neccesary leave feedback.
```

Comment:
Typo - 'go OVER all changes'

Reply:
@rrasmuss4200 Oops, I didn't get an email notification for any of your comments so I did not see them until now. I'll give it another read over and fix the typos. Also I'll add a section for drafting PR's. Thanks for the feedback!